src/app/mesh/qgsmeshtimeformatdialog.cpp

```diff
@@ -54,7 +58,11 @@ void QgsMeshTimeFormatDialog::loadSettings()
     mUseTimeComboBox->setCurrentIndex( 1 );
   }
 
-  mReferenceDateTimeEdit->setDateTime( settings.absoluteTimeReferenceTime() );
+  if ( settings.absoluteTimeReferenceTime().isValid() ) //set the reference time, if not valid, set the current date + time 00:00:00
```

Comment:
if you can move comment to separate line (line too wide), add 3rd person ("sets") and whitespace after //